music/cmd/sidecar_cmds.go

```
@@ -0,0 +1,279 @@
+// }
+
+var SidecarCmd = &cobra.Command{
+	Use:   "sidecar",
+	Short: "Prefix command, not usable directly",
+}
+
+var sidecarLocateCmd = &cobra.Command{
+	Use:   "locate",
+	Short: "Locate a sidecar, given its identity and method",
+	Run: func(cmd *cobra.Command, args []string) {
+		PrepArgs("identity", "method")
```

golangci-lint (1.62.2)

Error return value is not checked

(errcheck)